vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -47,9 +47,15 @@ namespace Microsoft.ReactNative
     void ReplaceChild(XAML_NAMESPACE.FrameworkElement parent, XAML_NAMESPACE.UIElement oldChild, XAML_NAMESPACE.UIElement newChild);
   }
 
-  [webhosthidden]
+  [webhosthidden] DOC_STRING("Enables a view manager to be responsible for its own layout and sizing.")
   interface IViewManagerRequiresNativeLayout
   {
     Boolean RequiresNativeLayout { get; };
   }
+
+  [webhosthidden]
+  DOC_STRING("Enables a view manager to dynamically decide what kind of view to create depending on the properties passed to it.")
```

Comment:
While it can be used this way, perhaps something like:

"Enables a view manager to create views using the property values passed to the manager at creation time."

It _could_ be used to change what kind view is returned, but more generically it could be used just to set up the view before it's added to the tree.

Reply:
fair point, i'll update it